.gitignore

```diff
@@ -102,4 +102,9 @@ __debug_bin
 .envrc
 
 # Develocity
+.mvn/.gradle-enterprise/
 .mvn/.develocity/
+
+# rat
+filtered_rat.txt
+rat.txt
```

Comment:
Why are these being added?

Reply:
They get created by the `release audit tool` when I run pre-commit 
As they're not part of the project, I assume I'm not meant to commit them, so i ignored them